pkg/release/release.go

```diff
@@ -36,6 +36,46 @@ const (
 	ocAdmReleaseInfo     = "oc adm release info quay.io/openshift-release-dev/ocp-release:%s-%s -o json"
 )
 
+type OSInterface interface {
```

Comment:
Nice:)
But just for reusability, maybe worth extracting to another pkg (e.g. pkg/fileutil). So we could use it nicely on other places.
Also, about the naming, maybe something like this instead: https://go.dev/talks/2012/10things.slide#8

Reply:
we have several packages involved in this. I think I can do that but in another PR since it will alter several pkgs.

Reply:
ok, so maybe just rename it. The naming here looks nice https://go.dev/talks/2012/10things.slide#8

Reply:
so just `osFS`?

Reply:
done

Reply:
@danielerez: Added `OSInterface` in https://github.com/openshift/appliance/pull/120
This PR is rebased on top of it.

Reply:
now rebased on master.